src/Post/Features/SettingsPage.php

```diff
@@ -15,76 +17,23 @@
  */
 trait SettingsPage {
```

Comment:
Yeah that's why I renamed it, but you make a good point about just referencing by its fully qualified name!